config/samples/data_v1alpha1_eacruntime.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: data.fluid.io/v1alpha1
```

Comment:
The file name should be changed to config/samples/data_v1alpha1_efcruntime.yaml

Reply:
fix